src/pages/ReportParticipantDetailsPage.tsx

```diff
@@ -0,0 +1,153 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import Avatar from '@components/Avatar';
+import Button from '@components/Button';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import MenuItem from '@components/MenuItem';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Report from '@libs/actions/Report';
+import * as UserUtils from '@libs/UserUtils';
+import Navigation from '@navigation/Navigation';
+import type {ParticipantsNavigatorParamList} from '@navigation/types';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import type {PersonalDetails, PersonalDetailsList} from '@src/types/onyx';
+import * as ReportUtils from '@libs/ReportUtils';
+import withReportOrNotFound from './home/report/withReportOrNotFound';
+import type {WithReportOrNotFoundProps} from './home/report/withReportOrNotFound';
+
+type ReportParticipantDetailsOnyxProps = {
+    /** Personal details of all users */
+    personalDetails: OnyxEntry<PersonalDetailsList>;
+};
+
+type ReportParticipantDetailsPageProps = WithReportOrNotFoundProps &
+    StackScreenProps<ParticipantsNavigatorParamList, typeof SCREENS.REPORT_PARTICIPANTS.DETAILS> & ReportParticipantDetailsOnyxProps;
+
+function ReportParticipantDetails({personalDetails, report, route}: ReportParticipantDetailsPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const StyleUtils = useStyleUtils();
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
+
+    const [isRemoveMemberConfirmModalVisible, setIsRemoveMemberConfirmModalVisible] = React.useState(false);
+
+    const accountID = Number(route.params.accountID);
```

Comment:
I am pretty stumped on what this is complaining about. @roryabraham might have some idea.

I can fix it by manually adding a `route.params.accountID` type, but unsure if we want to do it that way.

Is it something obvious? 😄

Reply:
This is caused by the `WithReportOrNotFoundOnyxProps` route type. Didn't check in depth but looks we faced similar issue in `PrivateNotesEditPage` where we fixed that by extending the route type i.e.:

```js
type WithReportOrNotFoundProps = WithReportOrNotFoundOnyxProps & {
    route:
        | RouteProp<ParticipantsNavigatorParamList, typeof SCREENS.REPORT_PARTICIPANTS.DETAILS>
        | RouteProp<PrivateNotesNavigatorParamList, typeof SCREENS.PRIVATE_NOTES.EDIT>
        | RouteProp<ReportDescriptionNavigatorParamList, typeof SCREENS.REPORT_DESCRIPTION_ROOT>;
```

Reply:
Not sure if I applied this suggestion correctly, but seems something else is broken now.

Reply:
Add `reportID` in `ParticipantsNavigatorParamList` 

```js
    [SCREENS.REPORT_PARTICIPANTS.DETAILS]: {
        reportID: string;
        accountID: string;
    };
    [SCREENS.REPORT_PARTICIPANTS.ROLE]: {
        reportID: string;
        accountID: string;
    };
```

Reply:
Ah wow. So, even if it's not used we still need it? Interesting...

Reply:
Ohhhh because it's used by `withReportOrNotFound()`. Not a fan of this complexity 😄